### ISSUE TITLE
Comments: How to detect if CSS transforms are supported on SVG

### ISSUE THREAD

**Issue by eprev** [eprev/eprev.org#15]
This issue is reserved for comments on [How to detect if CSS transforms are supported on SVG](https://eprev.org/2017/01/05/how-to-detect-if-css-transforms-are-supported-on-svg/).

**Comment by eli-crow** [comment]
Thanks for the article! Saved my butt! The Object.assign call should be replaced by individual property assignments for testing on older browsers (IE11-).

**Comment by chrismbarr** [comment]
For me this currently seems broken.

- Chrome 72 returns `false` (this is ❌ incorrect)
- Firefox 65 returns `true` (this is ✔️ correct)
- Edge 18 returns `true` (this is ❌ incorrect)
- IE throws an error because it doesn't support `Object.assign` ⚠️

**Comment by eprev** [comment]
Hey Chris, thanks for your comment.

Just checked https://jsfiddle.net/w9hrzotf/ in Chrome 72 and it prints `true`, which is correct.

As for Edge 18, according to [platform status](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/supportcsstransformsonsvg/) support for CSS transforms was added in Edge 17, this makes `true` correct.

**Comment by chrismbarr** [comment]
I am running Chrome 72 and it prints `false` for me

![image](https://user-images.githubusercontent.com/463685/53423295-1b348e80-39af-11e9-92da-8182ca51ca40.png)

**Comment by chrismbarr** [comment]
Just had a thought... I am running a hiDPI display on my laptop, and perhaps the 1px size doesn't work well here since that's actually about 1/4 of a "real pixel" and needs a larger margin of error.

I've updated the code to this, and I now get `true` printed out in  Chrome, Firefox, and Edge, and `false` in IE 11
https://jsfiddle.net/zwsnL35c/4/

**Comment by eprev** [comment]
Good catch, I managed to get `false` when the page was 125% of the original size. Crazy.

**Comment by chrismbarr** [comment]
Cool!  Do my changes still work correctly for a standard display?  I don't have access to one right now.

The web is a lot harder now that `1px !== 1px`

**Comment by chrismbarr** [comment]
So I guess I'm just having issues with animating SVG elements in Edge, which is why I was saying it was reporting things incorrectly.  My animations currently work in Chroms & Firefox, but not in Edge.  I am animating a `clip-path` and `transform: translate(x, y)` on SVG `<path>` elements and nothing is happening.  opacity animations work in edge, but that's about all.

**Comment by eprev** [comment]
Sorry, not familiar with it. As a rule of thumb, if you have a code sample that works in Chrome and FF as expected, and doesn’t in Edge, I’d suggest to check if the issue is known or report it https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/?page=1
